### ISSUE TITLE
Debate Module: search clear after enter

### ISSUE THREAD

**Issue by hmkay** [liqd/adhocracy-plus#336]
**URL:** https://aplus-dev.liqd.net/teststadt/subjects/2020-00170/
**user:** admin
**expected behaviour:** I can search with "Enter" and then delete the entry because the cross shows immerdiately
**behaviour:** the cross (delete) shows only after I clicked somewhere else if I typed in the search word and pressed "enter"
**important screensize:**
**device & browser:** 
**Comment/Question:** low prio.

**Comment by phillimorland** [comment]
@hmkay if you search with enter you can delete entry instantly anyway because your text cursor is still within the box so you just press cmd delete, this is default behavior so altering it might be tricky and as its outside of expected user actions i'll put a low prio on it. If it's not then please remove! :)

**Comment by hmkay** [comment]
@phillimorland : Makes sense. Probably just close it, actually. I think this does not need any work! 


![mic drop](https://user-images.githubusercontent.com/30862422/74748474-dc0ae780-5268-11ea-80ed-ab3730beca1f.gif)

**Comment by phillimorland** [comment]
NIce! i'll leave it open as it's something @mcastro-lqd also pointed out, we should try keep it in mind when we do further user testing

NOTE TO MYSELF: maybe we can activate clear btn as soon as any keydown?!

**Comment by lthueer** [comment]
@phillimorland can this one be closed or would you like to keep it open as a reminder?
